packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/index.tsx

```diff
@@ -131,18 +131,18 @@ export const SubscriptionUpgrade = ({
       <Header {...{ profile }} />
       <div className="main-content">
         <div
-          className="product-payment product-upgrade rounded-b-lg rounded-t-none"
+          className="page-title-container"
+          data-testid="subscription-update-heading"
+        >
+          <Localized id="product-plan-change-heading">
+            <h2 className="page-title">Review your change</h2>
+          </Localized>
+          <p className="subheading"></p>
+        </div>
+        <div
+          className="component-card product-payment product-upgrade rounded-b-lg rounded-t-none "
```

Comment:
`component-card` includes `rounded-b-lg` so this can be updated to `component-card product-payment product-upgrade rounded-t-none`

Let's remove `.subscription-upgrade-heading` from the stylesheet as well.